platforms/documentation/docs/src/docs/release/notes.md

```diff
@@ -16,23 +16,24 @@ This release makes [Configuration Cache](#config-cache) the preferred execution
 
 Gradle @version@ uses [Kotlin 2](#kotlin-2) and [Groovy 4](#groovy-4), and adopts [Semantic Versioning](#sem-ver) (SemVer) with version numbers in the format `MAJOR.MINOR.PATCH`.
 
-It also introduces several improvements for [build authors](#build-authoring), including much better Kotlin DSL script compilation performance, updates to the Gradle API, and a new type for Dependency Graph root nodes that allows detached configurations to resolve dependencies that reference their project.
+It also introduces several improvements for [build authors](#build-authoring), including much better Kotlin DSL script compilation avoidance, updates to the Gradle API, reproducible archive outputs, and a new dependency graph root type that allows detached configurations to resolve project dependencies.
 
 Gradle @version@ includes numerous bug fixes and general improvements.
 As a major release, it also introduces changes to deprecated APIs and behaviors.
 For details on what has been removed or updated, refer to the [Gradle 8.x upgrade guide](userguide/upgrading_version_8.html).
 
 We would like to thank the following community members for their contributions to this release of Gradle:
 [Aaron Matthis](https://github.com/rapus95),
-[Adam S](https://github.com/aSemy),
 [Adam E](https://github.com/adam-enko),
+[Adam S](https://github.com/aSemy),
 [Björn Kautler](https://github.com/Vampire),
 [Daniel Lacasse](https://github.com/lacasseio),
-[Ghost](https://github.com/ghost),
 [Eng Zer Jun](https://github.com/Juneezee),
 [EunHyunsu](https://github.com/ehs208),
 [FlorianMichael](https://github.com/FlorianMichael),
 [Francisco Prieto](https://github.com/priettt),
+[Gaëtan Muller](https://github.com/MGaetan89),
+[Ghost](https://github.com/ghost),
```

Comment:
Isn't ghost a "placeholder" account for accounts that have been deleted? Does it make sense to mention it?